maskrcnn_benchmark/structures/segmentation_mask.py

```diff
@@ -130,7 +177,8 @@ def convert(self, mode):
             )
             rle = mask_utils.merge(rles)
             mask = mask_utils.decode(rle)
-            mask = torch.from_numpy(mask)
+            mask = np.sum(mask, axis=2)
```

Comment:
Doesn't this have problems with the rest of the code? Like during training of Mask R-CNN, we call the convert function and that expects a tensor of a particular number of dimensions. I'm afraid this will break the code there.

Reply:
I think I'm missing something here, but how this won't be a breaking change compared to the previous code, as this is used in [here](https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/maskrcnn_benchmark/modeling/roi_heads/mask_head/loss.py#L40) to compute the loss?

Reply:
I mimics the code in Detectron ([here](https://github.com/facebookresearch/Detectron/blob/master/detectron/utils/segms.py#L84)). I didn't test it using polygons format. If this is wrong, I will remove it. I'm a little confused about why `sum` is not needed here.

Reply:
Well, this was working before, so this will probably crash now :-)
I don't remember all the details now and I'd need to check the difference. But maybe you can have a look at the code and run it to see what it gives?

Reply:
I just test the code on my RLE format dataset. It will take me some time to test if on coco.